src/main/java/uk/gov/ons/ctp/response/sample/ingest/CsvIngesterSocial.java

```diff
@@ -69,6 +69,8 @@ public SampleSummary ingest(final SampleSummary sampleSummary, final MultipartFi
     private SocialSampleUnit parseLine(CSVRecord line) throws CTPException{
         SocialSampleUnit sampleUnit = new SocialSampleUnit();
         sampleUnit.setAttributes(line.toMap());
+        String ref = (String) line.toMap().get("Reference");
+        sampleUnit.setSampleUnitRef(ref);
```

Comment:
Is casting to String necessary here? Also I don't think any of the tests check the sampleUnitRef is set correctly here.